platform/android/export/export_plugin.cpp

```diff
@@ -3159,6 +3153,18 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 		// Write an AndroidManifest.xml file into the Gradle project directory.
 		_write_tmp_manifest(p_preset, p_give_internet, p_debug);
 
+		// Enable or disable Monochrome icon in Gradle build.
+		String icon_xml = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+						  "<adaptive-icon xmlns:android=\"http://schemas.android.com/apk/res/android\">\n"
+						  "    <background android:drawable=\"@mipmap/icon_background\"/>\n"
+						  "    <foreground android:drawable=\"@mipmap/icon_foreground\"/>\n";
+		if (monochrome.is_valid() && !monochrome->is_empty()) {
+			icon_xml += "    <monochrome android:drawable=\"@mipmap/icon_monochrome\"/>\n";
+		}
+		icon_xml += "</adaptive-icon>";
+		const String icon_xml_path = gradle_build_directory.path_join("res/mipmap-anydpi-v26/icon.xml");
```

Comment:
Let's store the relative paths for `icon.xml` and `themed_icon.xml` as constants.